app/src/main/java/com/capstone/weatherapp/SingleCityResponse.kt

```diff
@@ -14,7 +14,25 @@ data class SingleCityResponse(
     val id: Int,
     val name: String,
     val cod: Int
-)
+) {
+    fun convertToSingleCityCache(singleCity: SingleCityResponse) : SingleCityCache {
```

Comment:
We can edit this to not take in any parameters and just use the class level data (found above). So, when you wanna call this, you can just do singleCityResponse.convertToSingleCityCache() instead of having to pass in an unnecessary parameter